src/pages/home.js

```diff
@@ -205,9 +206,18 @@ export default function Home() {
       for (const [relativePath, file] of Object.entries(imagesFolder.files)) {
         if (!relativePath.endsWith('/')) {
           const fileName = relativePath.split('/').pop();
-          const blob = await file.async('blob');
-          newImageFiles[fileName] = blob;
-          addImageToExport(fileName, blob);
+          let blob;
```

Comment:
1. Could we try to refactor and extract `importSource` to make it a pure function and move it into an individual module, for readability of home.js?
2. Could we also have a `toBlob` function as private function in the new module I mentioned above? It could make it more readable by early return.